package.json

```diff
@@ -74,6 +74,7 @@
 		"date-fns": "^2.11.1",
 		"date-fns-tz": "^1.0.10",
 		"emotion-theming": "^10.0.27",
+		"exenv": "^1.2.2",
```

Comment:
Added this since we are already using that indirectly via Chakra: https://github.com/chakra-ui/chakra-ui/blob/master/packages/chakra-ui/package.json#L31

Reply:
LOLZ I added a function with the exact same name for the toaster
it's likely the exact same logic too, since I think that package is a duplication of internal logic from React

Reply:
got tricked by chakra: https://github.com/chakra-ui/chakra-ui/blob/master/packages/chakra-ui/src/Modal/index.js#L22
***
but followed your approach, which is less dependent on external stuff